test/test_cgi_server.rb

```diff
@@ -0,0 +1,140 @@
+# coding: utf-8
+# frozen_string_literal: false
+
+require 'test/unit'
+require 'webrick'
+require_relative 'webrick_testing'
+require 'tempfile'
+require "xmlrpc/server"
+require 'xmlrpc/client'
+
+class Test_CGIServer < Test::Unit::TestCase
+  include WEBrick_Testing
+
+  def setup_http_server_option(use_ssl)
+    option = {
+      :BindAddress => "localhost",
+      :Port => 0,
+      :SSLEnable => use_ssl,
+    }
+    if use_ssl
+      require 'webrick/https'
+      option.update(
+        :SSLVerifyClient => ::OpenSSL::SSL::VERIFY_NONE,
+        :SSLCertName => []
+      )
+    end
+
+    option
+  end
+
+  def test_client_server
+    omit("The CGI file does not work on Windows") if RUBY_PLATFORM =~ /(mswin|mingw)/
```

Comment:
We can use `Gem.win_platform?`:

```suggestion
    omit("The CGI file does not work on Windows") if Gem.win_platform?
```